src/main/proto/wfa/measurement/internal/kingdom/differential_privacy.proto

```diff
@@ -0,0 +1,30 @@
+// Copyright 2021 The Cross-Media Measurement Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+syntax = "proto3";
+
+package wfa.measurement.internal.kingdom;
+
+option java_package = "org.wfanet.measurement.internal.kingdom";
+option java_multiple_files = true;
+
+// Parameters for differential privacy (DP).
+//
+// For detail, refer to "Dwork, C. and Roth, A., 2014. The algorithmic
+// foundations of differential privacy. Foundations and Trends in Theoretical
+// Computer Science, 9(3-4), pp.211-407."
+message DifferentialPrivacyParams {
```

Comment:
I am wondering if we should define such proto in the common repo. They are unlikely to change in the future.

I already created one in any-sketch and is referring to it inside the duchy. We could move it to the common repo and just use it everywhere.. 
 https://github.com/world-federation-of-advertisers/any-sketch/blob/bd54716d9c76664df363c0b16337c09443074b09/src/main/proto/wfa/common/noise_parameters.proto